### PR TITLE
created parser for jama network

### PR DESCRIPTION
**Pull request by librarywebchic**

### CODE REVIEW

**Comment by wilmouths** [comment]
Hello @librarywebchic  and @sdduffy
Log processing for Jama is already handled with the [Silverchair](https://github.com/ezpaarse-project/ezpaarse-platforms/tree/master/silverchair) parser.
Can you instead add the new “SEARCH” usecase to the existing Silverchair parser?

**Comment by librarywebchic** [comment]
I've updated this pull request to change the silverchair parser to include search and capture the publisher info

**Comment by wilmouths** [comment]
Hello @librarywebchic,

thank you for this update, however, the Jama Network parser is already present in the Silverchair parser, should it be maintained in Silverchair or is it necessary to make a new parser for this platform?

A large part of the team is on vacation, and needing advice from my colleagues, we will process your PR in early September.

**Comment by wilmouths** [comment]
Hello @librarywebchic ,

I see that you have modified the PR to modify the Silverchair parser, but I also see that the JAMA Network parser is also present in this PR, can you redo an exclusive PR for Silverchair, and validate me the cancellation of the JAMA Network parser creation ?

**Comment by librarywebchic** [comment]
I've removed the JAMA parser code

File: silverchair/parser.js [hunk 1]
```diff
@@ -18,6 +20,16 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
 
   let match;
 
+  if (host.includes('jamanetwork')) {
+    result.db_id = 'jama';
```
Comment (wilmouths) [inline silverchair/parser.js:24]: Prefer to use ``publisher_name`` instead of ``db_id`` in this case.

File: silverchair/parser.js [hunk 2]
```diff
@@ -18,6 +20,16 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
...
+  } else if (host.includes('asmedigitalcollection')) {
+    result.db_id = 'asmedigitalcollection';
+  } else if (host.includes('oup')) {
+    result.db_id = 'oup';
+  } else if (host.includes('gsw') | host.includes('geoscienceworld')) {
```
Comment (wilmouths) [inline silverchair/parser.js:29]: Prefer use ``||``  instead of ``|`` for an ``OR`` condition

**Pull request merged**